cpt/requirements.txt

```diff
@@ -1,3 +1,3 @@
-six>=1.10.0, <1.13.0
+six>=1.10.0, <1.15.0
 conan>=1.7.0, <1.23.0
-tabulate>=0.8.6
+tabulate>=0.8.0, <0.9.0
```

Comment:
what if I have tabulate > 0.9.0 on my machine? will it fail to install? same for six > 1.15.0?

Reply:
Yes, but the latest version available is 0.8.4 I think